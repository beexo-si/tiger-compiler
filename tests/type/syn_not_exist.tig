let
    type a = int
    type a = asdf
in
    0
end
