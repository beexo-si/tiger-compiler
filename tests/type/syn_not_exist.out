Fail: Error -- lÃ­nea 3: Tipo inexistente (asdf)

